src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentSelectionBase.cs

```diff
@@ -200,5 +202,26 @@ private static Document Format(ICommentSelectionService service, ITextSnapshot s
             var textSpans = changes.SelectAsArray(change => change.Span.ToTextSpan());
             return service.FormatAsync(document, textSpans, cancellationToken).WaitAndGetResult(cancellationToken);
         }
+
+        /// <summary>
+        /// Given a set of lines, find the minimum indent of all of the non-blank, non-whitespace lines.
+        /// </summary>
+        protected static int DetermineSmallestIndent(
+            SnapshotSpan span, ITextSnapshotLine firstLine, ITextSnapshotLine lastLine)
+        {
+            // TODO: This breaks if you have mixed tabs/spaces, and/or tabsize != indentsize.
```

Comment:
that seems unfortunate.  we have helpers for this purpose.  can we not use those here?

Reply:
oh, this was just a move.  n/m